pages/about/policies/antibullying.html

```diff
@@ -1 +1,401 @@
-antibullying.html
\ No newline at end of file
+<h1>Mount Sion CBS Secondary School Anti-Bullying Policy</h1>
+<ol>
+<li>
+<p>In accordance with the requirements of the Education (Welfare) Act 2000 and the code of behaviour guidelines issued by the NEWB, the Board of Management of Mount Sion CBS Secondary School has adopted the following anti-bullying policy within the framework of the school&rsquo;s overall code of behaviour. This policy fully complies with the requirements of the <em>Anti-Bullying Procedures for Primary and Post-Primary Schools </em>which were published in September 2013.</p>
+</li>
+</ol>
+<ol start="2">
+<li>
+<p>The Board of Management recognises the very serious nature of bullying and the negative impact that it can have on the lives of pupils and is therefore fully committed to the following key principles of best practice in preventing and tackling bullying behaviour:</p>
+</li>
+</ol>
+<ul>
+<li>
+<p>A positive school culture and climate which</p>
+<ul>
+<li>
+<p>is welcoming of difference and diversity and is based on inclusivity</p>
```

Comment:
@evanhearne The only suggestion I have is that you should tidy up the html. That html will read fine if you put the text in the 'li'

Reply:
OK, thank you, :)

Reply:
@denispop9  I'm going to leave it as it for now as it doesn't affect the functionality of the document.

Reply:
@evanhearne agreed it doesn't affect the functionality but one of the items on the checklist in the description is `my code follows the style guides of this project`. As a team member @denispop9 has identified some code styling improvements you could make here.

Reply:
OK, @laurafitzgerald @denispop9 all changed requested have been completed.